eclipse-collections-code-generator/src/main/resources/impl/list/mutable/primitiveArrayList.stg

```diff
@@ -109,7 +109,6 @@ public class <name>ArrayList extends Abstract<name>Iterable
 
     public void clear()
     {
-        Arrays.fill(this.items, <(zero.(type))>);
```

Comment:
Not sure how I feel about this change. The difference is probably only visible through reflection. I still think I'd prefer using the 4-arg version of `Arrays.fill` here too.